### PR TITLE
Adding core 6.1 to the suite

### PR DESCRIPTION
**Pull request by andreasohlund**
Plenty of manual tinkering :(

Wonder if there is a better way to do this

cc @timbussmann @dbelcham

### CODE REVIEW

**Comment by andreasohlund** [comment]
@Particular/nservicebus-maintainers please review and merge

File: src/Version_6_1/PubSub/EventHandler.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    IHandleMessages<Version_4_2.Messages.MyEvent>,
+    IHandleMessages<Version_4_3.Messages.MyEvent>,
+    IHandleMessages<Version_4_4.Messages.MyEvent>,
+    IHandleMessages<Version_4_5.Messages.MyEvent>,
+    IHandleMessages<Version_4_6.Messages.MyEvent>,
+    IHandleMessages<Version_4_7.Messages.MyEvent>,
+    IHandleMessages<Version_5_0.Messages.MyEvent>,
+    IHandleMessages<Version_5_1.Messages.MyEvent>,
+    IHandleMessages<Version_5_2.Messages.MyEvent>,
+    IHandleMessages<Version_6_0.Messages.MyEvent>
```
Comment (danielmarbach) [inline src/Version_6_1/PubSub/EventHandler.cs:17]: Does this need ` IHandleMessages<Version_6_1.Messages.MyEvent>`?

File: src/Version_6_1/SendReturn/SendReturnInitiator.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
+﻿using System.Threading.Tasks;
+using CommonMessages;
+using NServiceBus;
+
+public static class SendReturnInitiator
+{
+
+    public static void InitiateSendReturn(this IEndpointInstance bus)
+    {
+        foreach (var endpoint in EndpointNames.All)
```
Comment (danielmarbach) [inline src/Version_6_1/SendReturn/SendReturnInitiator.cs:10]: Should we make this async as well to align with other of those extensions? Does it need `Task.Run`?

File: src/Version_6_1/PubSub/EventHandler.cs (hunk 1 above)
Comment (andreasohlund) [reply src/Version_6_1/PubSub/EventHandler.cs:17]: Fixed

File: src/Version_6_1/SendReturn/SendReturnInitiator.cs (hunk 2 above)
Comment (andreasohlund) [reply src/Version_6_1/SendReturn/SendReturnInitiator.cs:10]: Done

**Comment by andreasohlund** [comment]
@danielmarbach should be good to go once green

**Review by danielmarbach — APPROVED** [review]

**Pull request merged**